File: Team 23 - Del 8 Handed in Version/Angular/AdminScreens/src/app/model/Exhibitions/exhibition-application.ts
import { User } from '../Users/user';
import { ApplicationStatus } from './application-status';
import { Exhibition } from './exhibition';

export class ExhibitionApplication {
  exhibitionApplicationID: number;
  exhibitionApplicationImage1: string;
  exhibitionApplicationImage2: string;
  exhibitionApplicationImage3: string;
  applicationDescription: string;

  exhibition: Exhibition;

  applicationStatus: ApplicationStatus;
  user: User;
}

File: Team 23 - Del 8 Handed in Version/BinaryBrainsAPI/BinaryBrainsAPI/Entities/Exhibitions/ExhibitionApplication.cs
using BinaryBrainsAPI.Entities.Images;
using BinaryBrainsAPI.Entities.Users;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace BinaryBrainsAPI.Entities.Exhibitions
{
    public class ExhibitionApplication
    {
        [Key]
        public int ExhibitionApplicationID { get; set; }
        public string ApplicationDescription { get; set; }
        public string ExhibitionApplicationImage1 { get; set; }
        public string ExhibitionApplicationImage2 { get; set; }
        public string ExhibitionApplicationImage3 { get; set; }

        [ForeignKey("ExhibitionID")]
        public int? ExhibitionID { get; set; }
        public Exhibition Exhibition { get; set; }

        [ForeignKey("ApplicationStatusID")]
        public int? ApplicationStatusID { get; set; }
        public ApplicationStatus ApplicationStatus { get; set; }

        [ForeignKey("UserID")]
        public int? UserID { get; set; }
        public User User { get; set; }
    }
}
